src/components/IdealImage/index.js

```diff
@@ -309,11 +309,19 @@ export default class IdealImage extends Component {
     }
   }
 
+  handleChange = inView => {
+    if (inView) {
+      this.onEnter()
+    } else {
+      this.onLeave()
+    }
+  }
+
   render() {
     const icon = this.props.getIcon(this.state)
     const message = this.props.getMessage(icon, this.state)
     return (
-      <Waypoint onEnter={this.onEnter} onLeave={this.onLeave}>
+      <Observer oChange={this.handleChange}>
```

Comment:
Should `oChange` be `onChange` on this line?

Reply:
Woops, pushing a fix thanks